app/main.py

```diff
@@ -1,3 +1,65 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: int or float) -> None:
+        self.km = km
+
+    def __str__(self) -> str:
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self) -> str:
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other: int or float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km += other
+            return self
+        else:
+            self.km += other.km
+            return self
+
+    def __mul__(self, other: int or float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km *= other
+            return self
+        else:
+            raise TypeError(f"{other} is not int")
+
+    def __truediv__(self, other: int or float) -> Distance:
+        if isinstance(other, (int, float)):
+            self.km /= other
+            self.km = round(self.km, 2)
+            return self
+        else:
+            raise TypeError(f"{other} is not int")
```

Comment:
The `__truediv__` method should handle division by zero to prevent runtime errors. Consider adding a check to raise a `ZeroDivisionError` if `other` is zero.